core/domain/domain.en.yml

```diff
@@ -592,6 +598,16 @@ responses:
         - title: No I'm done
           payload: "/done"
 
+  utter_ask_assess_to_answer_qa:
```

Comment:
There is already "utter_ask_assess_to_answer" which is also used in qa. Maybe rename this one to "uttter_ask_assess_to_answer_qa_as_fallback"? Or "utter_ask_assess_to_answer_with_offer_question". Or rename the other one too... Something to differentiate better.